package.json

```diff
@@ -78,7 +81,10 @@
     "to-boolean": "^1.0.0",
     "ts-node": "^3.3.0",
     "tsconfig-paths": "^2.2.0",
-    "typescript": "^2.4.2"
+    "typescript": "^2.4.2",
+    "url-parse": "^1.2.0",
+    "util": "^0.10.3",
+    "utill": "^0.1.3"
```

Comment:
What are these two for? Can't see them used in this PR.

Reply:
removes utill as I don't see it used any where but util is used for "isUndefined" validator check

Reply:
👍